src/sql/base/browser/ui/panel/panel.component.ts

```diff
@@ -328,15 +332,33 @@ export class PanelComponent extends Disposable implements IThemable {
 			this.focusPreviousTab();
 			eventHandled = true;
 		}
-
 		if (eventHandled) {
 			event.preventDefault();
 			event.stopPropagation();
 		}
 	}
 
+	/**
+	 * Making the selected tab header focusable when the focus leaves the tab header div.
```

Comment:
Generally this kind of additional context would go in a separate comment within the method - the doc comments should be shorter since they'll show up in the tooltip description and should usually just cover a description of what it does/what it's for. More detailed information about why something was done or specific implementation details are better done elsewhere so they don't clutter it up too much.